Scripts/NuGet/EdFi.Installer.WebApi/Install-EdFiOdsWebApi.psm1

```diff
@@ -365,74 +374,33 @@ function Get-WebApiPackage {
-        [hashtable]
-        [Parameter(Mandatory=$true)]
-        $Config
-    )
-    Invoke-Task -Name ($MyInvocation.MyCommand.Name) -Task {
-        $transformParams = @{
-            sourceFile = "$($Config.PackageDirectory)/Web.Base.config"
-            transformFile = "$($Config.PackageDirectory)/Web.Release.config"
-            destinationFile = "$($Config.PackageDirectory)/Web.config"
-            toolsPath = $Config.ToolsPath
-        }
-        Invoke-ConfigTransformation @transformParams
+        $Config.WebConfigLocation = $packageDir
     }
 }
-
-function Invoke-TransformWebConfigPostgreSQL {
-    [CmdletBinding()]
-    param (
-        [hashtable]
-        [Parameter(Mandatory=$true)]
-        $Config
-    )
-    Invoke-Task -Name ($MyInvocation.MyCommand.Name) -Task {
-        $transformParams = @{
-            sourceFile = "$($Config.PackageDirectory)/Web.config"
-            transformFile = "$($Config.PackageDirectory)/Web.Npgsql.config"
-            destinationFile = "$($Config.PackageDirectory)/Web.config"
-            toolsPath = $Config.ToolsPath
-        }
-        Invoke-ConfigTransformation @transformParams
-    }
-}
-
 function Invoke-TransformWebConfigAppSettings {
     [CmdletBinding()]
     param (
         [hashtable]
         [Parameter(Mandatory=$true)]
         $Config
     )
-    Invoke-Task -Name ($MyInvocation.MyCommand.Name) -Task {
-        $appSettings = @{
-            "apiStartup:type" = $Config.InstallType
-        }
-
-        if ($Config.WebApiFeatures) {
-            if ($Config.WebApiFeatures.ContainsKey("BearerTokenTimeoutMinutes")) {
-                $appSettings.BearerTokenTimeoutMinutes = $Config.WebApiFeatures.BearerTokenTimeoutMinutes
-            }
-            if ($Config.WebApiFeatures.ContainsKey("ExcludedExtensionSources")) {
-                $appSettings.ExcludedExtensionSources = $Config.WebApiFeatures.ExcludedExtensionSources
-            }
-            if ($Config.WebApiFeatures.FeatureIsEnabled) {
-                $Config.WebApiFeatures.FeatureIsEnabled.GetEnumerator() | ForEach-Object {
-                    $appSettings["$($_.key):featureIsEnabled"] = $_.value.toString()
+   
+        Invoke-Task -Name ($MyInvocation.MyCommand.Name) -Task {
+            $settingsFile = Join-Path $Config.WebConfigLocation "appsettings.json"
+            $settings = Get-Content $settingsFile | ConvertFrom-Json | ConvertTo-Hashtable
+            $settings.ApiSettings.Mode = $Config.InstallType
+            if($Config.WebApiFeatures){
+                if ($Config.WebApiFeatures.ContainsKey("BearerTokenTimeoutMinutes")) {
+                    $settings.ApiSettings.BearerTokenTimeoutMinutes=$Config.WebApiFeatures.BearerTokenTimeoutMinutes
+                }
+                if($Config.WebApiFeatures.ContainsKey("ExcludedExtensionSources"))
+                {
+                    $settings.ApiSettings.ExcludedExtensionSources= $Config.WebApiFeatures.ExcludedExtensionSources
                 }
             }
```

Comment:
This isn't quite right yet. This looks very similar to the `$AppSettingsOverrides` parameter from the sandbox admin installer. Since you're merging the object below you shouldn't need any of this:
```suggestion
```

Reply:
We should also update the docs above to have the new appsettings format
https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/blob/8ca8a27439f56bb3b1e334cce932696f2970ecfb/Scripts/NuGet/EdFi.Installer.WebApi/Install-EdFiOdsWebApi.psm1#L134-L147